includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -591,12 +591,20 @@ static function( $html ) {
 					return $html;
 				}
 
+				$img_width  = $src[1] ? (int) $src[1] : 0;
+				$img_height = $src[2] ? (int) $src[2] : 0;
+
+				// If height is zero, bail.
+				if ( 0 === $img_height ) {
```

Comment:
Might as well do this as well:

```suggestion
				if ( 0 === $img_height || 0 === $img_width ) {
```